pkg/tests/only_run_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2019-2020 Cisco Systems, Inc and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/networkservicemesh/cloudtest/pkg/config"
+	"github.com/networkservicemesh/cloudtest/pkg/utils"
+
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	"github.com/networkservicemesh/cloudtest/pkg/commands"
+)
+
+func TestOnlyRun(t *testing.T) {
```

Comment:
Can we cover the case when passed the wrong test name?

Reply:
AFAIK, there is no detection for wrong test name in CloudTest

Reply:
Should we somehow notify the user that if `only-run` directive has the wrong test name?

Reply:
I guess,  a warning can be issued if a test specified in top-level 'only-run' list is not found in final test list in findTests().

Reply:
Can we throw an error in method `findTests` if tests for the `only-run` are not found? I think it can help the user to understand what happens faster than check all lines in cloudtest logs to find the warning.

Reply:
Sounds good.
I think it worth a dedicated issue.

Reply:
#26 it is